doc/accounts.md

```diff
@@ -0,0 +1,190 @@
+  virtual balance, it is denied.
+* The on-chain balance of any RPC responses such as the `WalletBalance` RPC is
+  always shown as `0`. A custodial/restricted user shouldn't be able to see what
+  on-chain balance is available to the node operator as an account can only
+  spend off-chain balances anyway.
+* The off-chain balance (e.g. the response returned by the `ChannelBalance` RPC)
+  always reflects the account's virtual balance and not the node's overall
+  channel balance (and any remote balances are always shown as `0`).
+* The list of active/pending/closed channels is always returned empty. The
+  custodial/restricted user should not need to care (or even know) about
+  channels and their internal workings.
+* The list of payments and invoices is filtered to only return payments/invoices
+  created or paid by the account.
+* Invoices created by an account are mapped to that account. If/when such a
+  mapped invoice is paid, the amount is credited to that account's virtual
+  balance.
+
+## Use cases
+
+The following (definitely non-exhaustive) list of use cases is made possible by
+the accounts system:
+ - The "Uncle Jim" model: The tech-savvy person of the family (e.g. "Uncle Jim")
+   operates a Lightning node. He manages the liquidity of the node and provides
+   the capital for the channels. He can onboard his family members by creating
+   an account, locking a macaroon to that account and then scanning a QR code
+   with an app like [Zeus](https://github.com/ZeusLN/zeus) on the family
+   member's smartphone.
+ - The "spend up to a certain amount automatically" model: A web user has a
+   browser extension like [Alby](https://getalby.com/) installed and wants to
+   allow that extension to pay invoices for paywalls automatically up to a
+   certain amount per month. That amount could be enforced by the account so the
+   browser extension doesn't have to keep track of its spending actions. And an
+   account can be shared between extensions installed in different browsers.
+ - The "allowance" model: A parent wants to give their child their allowance in
+   Lightning satoshis. They create an account over the allowance amount and top
+   up the account each week/month.
+
+## HOWTO
+
+This section describes how an account can be created and used.
+
+### Create the account
+
+The first thing that needs to be done is to create the account with its initial
+balance (and an optional expiry). This **needs to be done by the node
+operator**, meaning access to the `lit.macaroon` is required.
+
+Example:
+```shell
+$ litcli accounts create 50000
+
+{
+        "account": {
+                "id": "d64dbc31b28edf66",
+                "initial_balance": "50000",
+                "current_balance": "50000",
+                "last_update": "1652353332",
+                "expiration_date": "0"
+        }
+}
+```
+
+This created a new account (ID ``) with an initial balance of 50k satoshis and
+no expiration.
+
+### Bake a new macaroon for the user
```

Comment:
Related to my comment above: what if we did this for the user?

Reply:
Yeah, great idea. Did that.